res/values/strings.xml

```diff
@@ -712,4 +746,27 @@
 	<string name="ConversationFragment_quoted_message_not_found">Original message not found</string>
 	<string name="reply_privately">Reply privately</string>
 
+    <!-- Emoji picker and categories -->
+    <string name="emoji_search_results">Search results</string>
+    <string name="emoji_not_found">No emoji found</string>
+    <string name="emoji_recent">Recently Used</string>
+    <string name="emoji_people">People &amp; Body</string>
+    <string name="emoji_nature">Animals &amp; Nature</string>
+    <string name="emoji_foods">Food &amp; Drink</string>
+    <string name="emoji_activity">Activity</string>
+    <string name="emoji_places">Travel &amp; Places</string>
+    <string name="emoji_objects">Objects</string>
+    <string name="emoji_symbols">Symbols</string>
+    <string name="emoji_flags">Flags</string>
+
+    <!-- reasons for a disabled message composer -->
+    <string name="messaging_disabled_not_in_group">You are not in this group and cannot write therefore. To join, ask another member.</string>
```

Comment:
```suggestion
    <string name="messaging_disabled_not_in_group">You are not in this group. To join, ask another member.</string>
```
or
```suggestion
    <string name="messaging_disabled_not_in_group">You cannot write because you are not in this group. To join, ask another member.</string>
```